src/lib/schematics/nav/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.ts

```diff
@@ -53,6 +53,6 @@ import { Observable } from 'rxjs';
   changeDetection: ChangeDetectionStrategy.<%= changeDetection %><% } %>
 })
 export class <%= classify(name) %>Component {
-  isHandset: Observable<BreakpointState> = this.breakpointObserver.observe(Breakpoints.Handset);
+  isHandset$: Observable<BreakpointState> = this.breakpointObserver.observe(Breakpoints.Handset);
```

Comment:
Since all the usages of `isHandset` are checking the `matches`, it would be cleaner if we added a `.pipe(map(result => result.matches))` here. That way we can use `isHandset$ | async` in the templates, rather than `(isHandset$ | async)!.matches`.